### ISSUE TITLE
Effects Nubs are not on the entire image just partial

### ISSUE THREAD

**Issue by mohamedmansour** [mohamedmansour/my-hangouts-extension#10]
Try out an effect with a Nub and you will see they are only half the image. I think the width/height is misplaced somehow somewhere.

**Comment by jwill** [comment]
See pull request #65. It's fixed in that patch.

**Comment by mohamedmansour** [comment]
There is an issue where the nubs do not work now, lots of errors in console due to "left" not found.

```
 Uncaught TypeError: Cannot read property 'left' of null
```

I changed it (line 489) to:

```
 var offset = $('#canvasPreview').offset();
```

But I didn't look further if that is the correct fix, the nubs should be bounded within the canvas, for some reason, it is bounded to the entire frame.

**Comment by jwill** [comment]
Yep. That is the proper thing to do. The rebase clobbered that line. I knew about the constraint not being quite right but I preferred that to the way the nubs were.

**Comment by jwill** [comment]
Fixed in #66

**Comment by mohamedmansour** [comment]
Perfect, awesome :)
